LabExT/View/MainWindow/MainWindowView.py

```diff
@@ -62,21 +63,26 @@ def __init__(self, parent, menu_listener):
             command=self._menu_listener.client_setup_stages)
         self._movement.add_command(
             label="Configure Mover...",
-            command=self._menu_listener.client_setup_mover)
+            command=self._menu_listener.client_setup_mover,
+            state=NORMAL if self._menu_listener._experiment_manager.mover.has_connected_stages else DISABLED)
```

Comment:
Is this really true that one can only configure the mover if there are connected stages? The mover object is existing anyhow before any stages are connected, no?

I do see why we could simply disable configuring the mover for workflow reasons, to make it less confusing for people... So I have nothing against this check but I'm just curious.

Reply:
The current implementation requires connected stages. The mover wizard invokes e.g the following method https://github.com/LabExT/LabExT/blob/dev/LabExT/Movement/MoverNew.py#L431. This sets the property directly in the stage, so we need at least one connected stage.

On startup, when the mover gets initialised, we load these property only internally.